docker-for-mac/troubleshoot.md

```diff
@@ -6,77 +6,106 @@ redirect_from:
-team. For more information on our policy regarding personal data you can read
+team. For more information on our policy regarding personal data, see
 [how is personal data handled in Docker
 Desktop](https://docs.docker.com/docker-for-mac/faqs/#how-is-personal-data-handled-in-docker-desktop).
 
 ![Diagnostics & Feedback with
 ID](images/diagnose-feedback-id.png){:width="600px"}
 
-If you click **Report an issue**, this opens [Docker Desktop for Mac issues on
-GitHub](https://github.com/docker/for-mac/issues/) in your web browser in a
-"create new issue" template, to be completed before submission. Do not forget to
-copy/paste your diagnostic ID.
-
-![issue template](images/issues-template.png){:width="600px"}
+If you click **Report an issue**, this opens Docker Desktop [for Mac](https://github.com/docker/for-mac/issues/) issues on GitHub in your web browser in a "New issue" template. Add the details before submitting the issue. Do not forget to copy/paste your diagnostic ID.
 
 ### Diagnosing from the terminal
 
-On occasions it is useful to run the diagnostics yourself, for instance if
-Docker Desktop for Mac cannot start.
+In some cases, it is useful to run the diagnostics yourself, for instance if
+Docker Desktop cannot start.
 
-First locate the `com.docker.diagnose` tool.  If you installed Docker Desktop for Mac in
-the Applications directory, then it is
+First, locate the `com.docker.diagnose` tool.  If you have installed Docker Desktop in the Applications directory, then it is located at
 `/Applications/Docker.app/Contents/MacOS/com.docker.diagnose`.
 
-Then to create *and upload* diagnostics, run:
+To create *and upload* diagnostics, run:
 
 ```sh
 $ /Applications/Docker.app/Contents/MacOS/com.docker.diagnose gather -upload
 ```
+
 After the diagnostics have finished, you should have the following output,
 containing your diagnostics ID:
 
 ```sh
-Diagnostics Bundle: /tmp/B8CF8400-47B3-4068-ADA4-3BBDCE3985D9/20180726143610.zip
-Diagnostics ID:     B8CF8400-47B3-4068-ADA4-3BBDCE3985D9/20180726143610 (uploaded)
+Diagnostics Bundle: /tmp/B8CF8400-47B3-4068-ADA4-3BBDCE3985D9/20190726143610.zip
+Diagnostics ID:     B8CF8400-47B3-4068-ADA4-3BBDCE3985D9/20190726143610 (uploaded)
+Diagnostics Bundle: /tmp/BE9AFAAF-F68B-41D0-9D12-84760E6B8740/20190905152051.zip
+Diagnostics ID:     BE9AFAAF-F68B-41D0-9D12-84760E6B8740/20190905152051 (uploaded)
 ```
 
-The diagnostics ID (here B8CF8400-47B3-4068-ADA4-3BBDCE3985D9/20180726143610) is
-composed of your user ID (D1F48686-F045-4708-85E3-0635B729A596) and a timestamp
-(20180525-145051).  Be sure to provide us with the full diagnostics ID, not just
+The diagnostics ID (here BE9AFAAF-F68B-41D0-9D12-84760E6B8740/20190905152051) is
+composed of your user ID (BE9AFAAF-F68B-41D0-9D12-84760E6B8740) and a timestamp
+(20190905152051).  Be sure to provide us with the full diagnostics ID, not just
 the user ID.
 
 Don't hesitate browsing the content of these diagnostics:
```

Comment:
This is a bit confusing. I think it's the "don't hesitate" part. Suggest changing to "Be sure to view the diagnostic content in the following location"

Reply:
Agree. It sounds confusing. Here's the updated version:
_I've retained two sentences as it involves separate actions (Providing the diagnostics ID and viewing the diagnostic file)._

Ensure you provide the full diagnostics ID, and not just the user ID.

To view the content of the diagnostic file, run:

```sh
$ open /tmp/BE9AFAAF-F68B-41D0-9D12-84760E6B8740/20190905152051.zip
```